Code/ros2_files.cmake

```diff
@@ -4,69 +4,98 @@
 # SPDX-License-Identifier: Apache-2.0 OR MIT
 
 set(FILES
-    Source/Camera/CameraSensor.cpp
```

Comment:
Nitpick: Wasn't the old indentation correct? Also, it would be easier to track the difference if we keep it

Reply:
The old one got corrected by clang (perhaps overzealous to apply to .cmake)